source/Event.py

```diff
@@ -38,25 +39,32 @@ def max_name_length(self):
             max_length = len(p.name) if len(p.name) > max_length else max_length
         return max_length
 
-    def load_participants(self, csv_file: str):
+    def load_participants(self, csv_file: str, member_ability_csv_file: str):
         """
-        Loads participants from a CSV file.
+        Loads participants from an MSR Exported CSV file, then gets their possible work assignments from the private Member CSV file.
 
         Returns:
             list[Participant]: All parsed participants.
         """
+        member_ability_dictionary = {}
+        with open(member_ability_csv_file, newline="", encoding="utf-8") as member_ability_csv:
+            member_data = csv.DictReader(member_ability_csv)
+            for _id, row in enumerate(member_data):
+                member_ability_dictionary[row["id"]] = row
+
         participants = []
         with open(csv_file, newline="", encoding="utf-8-sig") as file:
             reader = csv.DictReader(file)
             for i, row in enumerate(reader):
+                member_ability_data = member_ability_dictionary.get(utils.get_formatted_member_number(row["Member #"]), {})
                 participant = Participant(
                     event=self,
                     id=i,
-                    name=row["name"],
-                    category_string=row["category"],
-                    novice=utils.parse_bool(row["novice"]),
+                    name=row["Name"],
+                    category_string=row["Class"] if row["Modifier"] in ["", "NOV"] else row["Modifier"], # Lemme know if this isn't pythonic, looks ugly
+                    novice=utils.parse_bool(row["Modifier"] == "NOV"),
```

Comment:
Line 64 looks good to me, feel free to remove the comment.

More broadly speaking - how customized is our MSR form? Is there a standard template, or does everyone just do their own thing? Implementation looks good but I'm working how we can make it as generic as reasonable (if it makes sense to in the first place).